### ISSUE TITLE
Issues and suggestion

### ISSUE THREAD

**Issue by isoz** [Muqsit/SkyWars-PocketMine#14]
Sometimes players go invisible and won't become visible until relog.
The $count is not very accurate, for example there's only 2 players left in a match, I kill the last player and it will say in chat "isoz killed XXX [2/12]".

A leaderboard would be very useful, something that stores how many times a player has won and display it with a floatingtext.

**Comment by isoz** [comment]
I believe the invisible problem is caused by something in the spectator, I disabled spectator and now no invisible problems.

**Comment by ghost** [comment]
Another thing is that chest refill doesn't work and some chests don't have items

**Comment by isoz** [comment]
Agree, when the round starts all chests are filled and well. But on the refill event some don't.
